dnsapi/dns_arvancdn

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env #!/bin/sh
+
+# Author: Mohammad Ali Sarbanha <sarbanha at yahoo dot com>
+# Repository: https://github.com/sarbanha/acme.sh-dnsapi-dns_arvancdn
+
+# export ARVAN_API_KEY="-----------"
+
+ARVAN_CDN_API="https://napi.arvancloud.com/cdn/4.0"
+
+#Usage: dns_arvancdn_add _acme-challenge.www.domain.com "XKrxpRBosdIKFzxW_CT3KLZNf6q0HG9i01zxXp5CPBs"
+dns_arvancdn_add() {
+
+  fulldomain=$1
+  challenge=$2
+  zone=${fulldomain:16}
```

Comment:
![image](https://user-images.githubusercontent.com/8305679/175794660-93d2f4a6-a88a-4f9f-9c74-730a16057f38.png)

Reply:
Thanks for the comments, I am about to update the new tested code.